Dockerfile

```diff
@@ -59,19 +58,18 @@ RUN apt-get update --assume-yes \
         zlib1g-dev \
         libsdl2-dev \
         mesa-common-dev \
-        libfreetype6-dev \
-        libpng12-dev \
-        libboost-thread-dev \
-        libboost-system-dev \
-        libboost-signals-dev \
-        libboost-serialization-dev \
-        libboost-regex-dev \
-        libboost-python-dev \
-        libboost-log-dev \
-        libboost-iostreams-dev \
-        libboost-locale-dev \
-        libboost-filesystem-dev \
-        libboost-date-time-dev \
-        libboost-test-dev \
+        libpng-dev \
+        libboost-thread1.62-dev \
+        libboost-system1.62-dev \
+        libboost-signals1.62-dev \
+        libboost-serialization1.62-dev \
+        libboost-regex1.62-dev \
+        libboost-python1.62-dev \
```

Comment:
Does this automatically install python 2 and 3 support for boost?

Reply:
Yes, it depends on both pythons.